src/components/Card.tsx

```diff
@@ -17,7 +17,20 @@ const Description = styled.p`
 
 const TopContent = styled.div``
 
-const Card = ({ emoji, title, description, children, className }) => (
+interface IProps {
```

Comment:
```suggestion
export interface IProps {
```
Just as a convention, to keep all components props interfaces exposed.